.travis.yml

```diff
@@ -3,8 +3,6 @@ node_js:
   - "4"
   - "5"
   - "6"
-script:
```

Comment:
I'd be OK with it if a comment to the Travis documentation was also added.
In the spirit of h5bp there should be pointers for people to learn about
what they're using.
On 17 May 2016 1:19 AM, "Hugo Giraudel" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/kaelig/sass-project-boilerplate/pull/1#discussion_r63370799>
> :
>
> > @@ -3,8 +3,6 @@ node_js:
> >    - "4"
> >    - "5"
> >    - "6"
> > -script:
>
> Maybe being a bit more verbose makes sense in a boilerplate?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaelig/sass-project-boilerplate/pull/1/files/4c6935f6f309b5617eadc1d7c7f49d23f64a9ede#r63370799>
>